src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/PlaceholderMethodSymbol.cs

```diff
@@ -166,6 +166,16 @@ bool Cci.ISignature.ReturnValueIsByRef
             get { return true; }
         }
 
+        // This should be inherited from the base class implementation, but it does not currently work with Nullable
```

Comment:
What? Why? Do we have a bug tracking this?

Reply:
Unknown if there is already a bug. I'll file a bug for this one if it's not corrected before merge.

Reply:
📝 I was not able to find a way to work around this bug, so this change is unavoidable if we want to annotate this file.

Reply:
What error are you getting?

Reply:
> C:\\dev\\roslyn\\src\\ExpressionEvaluator\\CSharp\\Source\\ExpressionCompiler\\Symbols\\PlaceholderMethodSymbol.cs(15,67): error CS0738: 'PlaceholderMethodSymbol' does not implement interface member 'ISignatu
re.RefCustomModifiers'. 'PlaceholderMethodSymbol.RefCustomModifiers' cannot implement 'ISignature.RefCustomModifiers' because it does not have the matching return type of 'ImmutableArray\<ICustomModif
ier>'.

Reply:
@AlekseyTs, any idea why this could be occuring?

Reply:
Lets make sure we have a bug on this (linked here), or back this file out of this PR.

Reply:
> or back this file out of this PR.

This pull request will not compile without this change.

Reply:
> any idea why this could be occuring?

In general, enabling nullability shouldn't cause errors. Please open an issue.

Reply:
@sharwell please create an issue and link in source. This is the approach we generally prefer on the compiler team.

Reply:
Done